src/main/java/com/startup/models/Teacher.java

```diff
@@ -0,0 +1,91 @@
+package com.startup.models;
+
+public class Teacher {
+
+    private long teacherId;
+    private String firstName;
+    private String lastName;
+    private String gender;
+    private int cnp;
```

Comment:
cnp should be a String type as it holds 13 digits, the maximum value for an int is 2147483647. Or if you want we can change it to long.